assets/src/dashboard/app/views/previewStory/index.js

```diff
@@ -65,6 +65,10 @@ const IframeContainer = styled.div`
   height: ${({ dimensions }) =>
     `${dimensions.height - CLOSE_BUTTON_SIZE.HEIGHT}px`};
   min-height: 90vh;
+
+  &:focus {
+    border: ${({ theme }) => theme.borders.bluePrimary};
```

Comment:
One thing I did notice - in the editor it looks like @littlemilkstudio used 2px for the outline - we should probably be consistent (https://github.com/google/web-stories-wp/pull/4248/files#diff-aa3e8cac28a490dabe690b3a4d9d5d81R58)

Reply:
yea I wish the dashboard and editor themes were more unified 😢

Reply:
we could always start making utils like Brittany did for the fonts that way it's all in one place:
`/src/component-library`
```
// pull this into either /edit-story or /dashboard:
export const focusable = theme => css`
   border: ${({ theme }) => theme.borders.bluePrimary};
`;

const SomeStyledComponent = styled.div`
  ${focusable}
`;
```
Honestly fine with leaving this as is for now cus I just started doing the 2px outline off of other code I saw, but something to consider when we start fine tuning the theme/component library structure more

Reply:
good call. i think the focus requirements on borders just got updated: https://www.w3.org/TR/2020/WD-WCAG22-20200811/#focus-appearance-enhanced I'll make this 2px and as we transition to the new design system we'll gain continuity.